openapi/__tests__/definitions.test.ts

```diff
@@ -232,6 +235,50 @@ const aPayee: Payee = { fiscal_code: anOrganizationFiscalCode };
 
 const aThirdPartyId = "aThirdPartyId" as NonEmptyString;
 
+describe("BaseEnrichedMessage definition", () => {
+  it("should decode a BaseEnrichedMessage", () => {
+    const aBaseEnrichedMessage = {
+      ...aMessageWithoutContent,
+      service_name: "aService",
+      organization_name: aService.organizationName,
+      message_title: "aTitle",
+    };
+
+    expect(
+      E.isRight(BaseEnrichedMessage.decode(aBaseEnrichedMessage))
+    ).toBe(true);
+  });
+});
+
+describe("EnrichedMessageWithOrganizationFiscalCode definition", () => {
+  it("should fail decoding a EnrichedMessageWithOrganizationFiscalCode if the CF is not provided", () => {
+    const aBaseEnrichedMessage = {
+      ...aMessageWithoutContent,
+      service_name: "aService",
+      organization_name: aService.organizationName,
+      message_title: "aTitle",
+    };
+
+    expect(
+      E.isLeft(EnrichedMessageWithOrganizationFiscalCode.decode(aBaseEnrichedMessage))
+    ).toBe(true);
+  });
+
+  it("should decode correctly a EnrichedMessageWithOrganizationFiscalCode if the CF is provided", () => {
+    const aBaseEnrichedMessage = {
```

Comment:
This is WithOrganizationFiscalCode.

Reply:
[62b5154](https://github.com/pagopa/io-functions-commons/pull/379/commits/62b51540b5922ddba87abde64c52e5e14ea42c57)